python/lsst/sims/photUtils/SignalToNoise.py

```diff
@@ -7,22 +7,45 @@
           "calcM5", "calcSkyCountsPerPixelForM5", "calcGamma", "calcSNR_m5",
           "calcAstrometricError", "magErrorFromSNR", "calcMagError_m5", "calcMagError_sed"]
 
+def FWHMeff2FWHMgeom(FWHMeff):
```

Comment:
You need to add these new methods to the __all__ at the top of the file so users can import them like

from lsst.sims.photUtils import FWHMeff2FWHMgeom